src/actions/guides/getting-started/installing.cr

```diff
@@ -20,6 +20,9 @@ class Guides::GettingStarted::Installing < GuideAction
     * [Crystal v0.30](https://crystal-lang.org/reference/installation/)
     * Postgres ([macOS](https://postgresapp.com)/[Others](https://wiki.postgresql.org/wiki/Detailed_installation_guides))
 
+    You will need additional dependencies if you want to test your frontend using LuckyFlow,
+    see the [_Testing HTML and Interactivity_ guide](#{Guides::Frontend::Testing.path}) for details.
```

Comment:
```suggestion
    see the [Testing HTML and Interactivity](#{Guides::Frontend::Testing.path}) guide for details.
```
Just for consistency, lets take off the underscore. I don't think we have any other links with italics.

Reply:
:+1: Done!